### PR TITLE
fix(theme): Fix the abandoned normal level in the theme file. close #17473

### LINKED ISSUE #17473: [Bug] 5.3.3版本使用主题文件出现警告信息
### Version

5.3.3

### Link to Minimal Reproduction

_No response_

### Steps to Reproduce

1.使用npm安装echarts；
2.使用示例默认的选项；
3.引入了macarons，shine主题；

### Current Behavior

在引入了shine主题文件后，在控制台出现警告信息：

> [ECharts] DEPRECATED: 'normal' hierarchy in iconStyle has been removed since 4.0. All style properties are configured in iconStyle directly now.
发现是theme/shine.js内toolbox配置项导致，并且macarons.js也有同样问题。
` toolbox: {
            iconStyle: {
                normal: {
                    borderColor: '#06467c'
                }
            }
        },`

### Expected Behavior

希望不要出现这个警告了:)

### Environment

```markdown
- OS:
- Browser:
- Framework:
```

### Any additional comments?

_No response_

### PR DESCRIPTION
**Pull request by Liangism**
## Brief Information

This pull request is in the type of:

- [x] bug fixing
- [ ] new feature
- [ ] others

## Document Info

One of the following should be checked.

- [x] This PR doesn't relate to document changes
- [ ] The document should be updated later
- [ ] The document changes have been made in apache/echarts-doc#xxx

### Merging options

- [ ] Please squash the commits into a single one when merging.

### Other information

### CODE REVIEW

File: theme/tool/option/map.js [hunk 1]
```diff
@@ -35,9 +35,7 @@ export default {
             map: 'china',
             showLegendSymbol: true,
             label: {
-                normal: {
-                    show: false
-                },
+                show: false,
                 emphasis: {
                     show: false
                 }
```
Comment (plainheart) [inline theme/tool/option/map.js:41]: `label.emphasis` should be changed to
```js
emphasis: {
  label: {
    // xxx
  }
}
```

The same below.

File: theme/tech-blue.js [hunk 2]
```diff
@@ -105,7 +105,7 @@
                 color: '#00aecd'
             },
             controlStyle: {
-                normal: { color: '#00aecd' },
+                color: '#00aecd',
                 emphasis: { color: '#00aecd' }
```
Comment (plainheart) [inline theme/tech-blue.js:109]: The definition of `emphasis` state of `controlStyle` should be changed to

```js
emphasis: {
  controlStyle: {
    color: '#00aecd'
  }
}
```

**Comment by plainheart** [comment]
Hi @Liangism, why did you close this PR? You can push new commits to the current branch to fix the issues.

**Comment by Liangism** [comment]
I submitted a PR again, and I modified some of the previous Theme files. :)


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "apache/echarts"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年8月8日(星期一) 上午10:08
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [apache/echarts] fix(theme): Fix the abandoned normal level in the theme file. close #17473 (PR #17476)





 
Hi @Liangism, why did you close this PR? You can push new commits to the current branch to fix the issues.

**Comment by Liangism** [comment]
Sorry, I ordered wrong




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Zhongxiang ***@***.***&gt;; 
发送时间: 2022年8月8日(星期一) 上午10:08
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [apache/echarts] fix(theme): Fix the abandoned normal level in the theme file. close #17473 (PR #17476)





 
Hi @Liangism, why did you close this PR? You can push new commits to the current branch to fix the issues.

**Pull request closed without merging**